docs/data-sources/system.md

```diff
@@ -7,69 +7,90 @@ description: |-
+* `fabric_mac` (fabricMAC) - (string) 
+* `inb_mgmt_addr` (inbMgmtAddr) - (string) The in-band management IPv4 address.
+* `inb_mgmt_addr6` (inbMgmtAddr6) - (string) In-band management IPv6 address.
+* `inb_mgmt_addr6_mask` (inbMgmtAddr6Mask) - (string) In-band management IPv6 address Mask.
+* `inb_mgmt_addr_mask` (inbMgmtAddrMask) - (string) In-band management IP address Mask.
+* `inb_mgmt_gateway` (inbMgmtGateway) - (string) In-band management IP Gateway.
+* `inb_mgmt_gateway6` (inbMgmtGateway6) - (string) In-band management IPv6 Gateway.
+* `last_reboot_time` (lastRebootTime) - (string) LastRebootTime for this system.
+* `last_reset_reason` (lastResetReason) - (string) LastResetReason for this system.
+* `lc_own` (lcOwn) - (string) A value that indicates how this object was created. For internal use only.
+* `mod_ts` (modTs) - (string) The time when this object was last modified.
+* `mode` (mode) - (string) Specifies if this system is configured in standalone mode or HA pair.
+* `mon_pol_dn` (monPolDn) - (string) The monitoring policy attached to this observable object.
+* `name` (name) - (string) The name of the System object.
+* `name_alias` (nameAlias) - (string) The name alias of the System object.
+* `node_type` (nodeType) - (string) Role of this system.
+* `oob_mgmt_addr` (oobMgmtAddr) - (string) The out-of-band management IPv4 address.
+* `oob_mgmt_addr6` (oobMgmtAddr6) - (string) Out-of-band management IPv6 address.
+* `oob_mgmt_addr6_mask` (oobMgmtAddr6Mask) - (string) Out-of-band management IPv6 address Mask.
+* `oob_mgmt_addr_mask` (oobMgmtAddrMask) - (string) Out-of-band management IP address Mask.
+* `oob_mgmt_gateway` (oobMgmtGateway) - (string) Out-of-band management IP Gateway.
+* `oob_mgmt_gateway6` (oobMgmtGateway6) - (string) Out-of-band management IPv6 Gateway.
+* `remote_network_id` (remoteNetworkId) - (string) Remote Network ID.
+* `remote_node` (remoteNode) - (string) Remote system.
+* `rl_auto_mode` (rlAutoMode) - (string) Remote Leaf Auto Mode.
+* `rl_group_id` (rlGroupId) - (string) Remote Leaf Site Identifier.
+* `rl_oper_pod_id` (rlOperPodId) - (string) Operational POD Idenitfier for RL Pod Redundancy.
+* `rl_routable_mode` (rlRoutableMode) - (string) Is RemoteLeaf Routable.
+* `rldirect_mode` (rldirectMode) - (string) Remote Leaf Direct Mode.
+* `role` (role) - (string) The system role type.
+* `serial` (serial) - (string) Serial Number of the system.
+* `server_type` (serverType) - (string) DhcpServerType.
+* `site_id` (siteId) - (string) Site Identifier.
+* `state` (state) - (string) Operational state of this system.
+* `system_up_time` (systemUpTime) - (string) The time (in seconds) since the system was booted.
+* `tep_pool` (tepPool) - (string) TepPool for this system.
+* `unicast_xr_ep_learn_disable` (unicastXrEpLearnDisable) - (string) Disable xrLeanrs.
```

Comment:
what does xr stand for in xrlearn?

Reply:
I looked it up and it is external routing..updated it accordingly